config/default.yml

```diff
@@ -501,6 +501,15 @@ Rails/SkipsModelValidations:
     - update_counters
   Whitelist: []
 
+Rails/TagInsteadOfContentTag:
+  Description: 'Use `tag` instead of `content_tag`.'
+  Reference:
+    - 'https://github.com/rails/rails/issues/25195'
+    - 'https://apidock.com/rails/ActionView/Helpers/TagHelper/content_tag'
```

Comment:
Can we link to the official docs instead?

Reply:
Thank you for your review!
I fixed and pushed it.